### ISSUE TITLE
Building delegate using Docker does not copy delegate includes to output tar

### ISSUE THREAD

**Issue by lablabla** [ARM-software/armnn#742]
I'm using the build-tool in docker to build ArmNN with its TFLite delegates, but when I look at the output tarball, it does not contain the delegates include files in it.

Looking at the `build-armnn.sh` file, it looks like something in the form of

```
  mkdir -p include/delegate
  cp "$SOURCE_DIR"/armnn/delegate/include/* include/delegate
```

is missing after the other include copy in [line 168](https://github.com/ARM-software/armnn/blob/branches/armnn_23_11/build-tool/scripts/build-armnn.sh#L168)

and similar for the classic/opaque include directories (or maybe only for these directories since the `delegate` directory points to the classic dir

Using 23.11 branch with the following docker build command
```
docker build --build-arg SETUP_ARGS="--target-arch=aarch64 --all" --build-arg BUILD_ARGS="--target-arch=aarch64 --all --neon-backend --cl-backend" --tag armnn:aarch64 --file docker/Dockerfile .
```

Am I missing something in using the build-tool and/or the delegates?

**Comment by nikraj01** [comment]
Hi @lablabla ,

Thanks for reporting the issue. I will have a look into this.

**Comment by John-ARM** [comment]
Hello,
This change has been made and will be included in the 24.05 release of Arm NN.
Thank you
John
